api/exception/tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
```

Comment:
question: why are we not using the existing `test` folder?

Reply:
I wanted to separate tests by package, but I realize it's "too soon" since I tracked this in our board.
I will move it under the common tests folder, thanks!

Reply:
I moved the tests under the common folder